### PR TITLE
Refactor/billing migration

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | no
| Breaking change? | yes
| Tickets          | Fix #MANAGER-5821
| License          | BSD 3-Clause

## Description

Migrate billing code to a standalone module

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report test#16807.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report build#16807.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#16807.1 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report test#16810.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report build#16842.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report test#16842.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report test#16845.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report test#16866.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✘

**Comment by frenautvh** [comment]
some conflicts to resolve,
also is it not possible to use git mv ? files are displayed as created/deleted instead of moved

**Comment by radireddy** [comment]
> some conflicts to resolve,
> also is it not possible to use git mv ? files are displayed as created/deleted instead of moved

I have used git move for billing migration. I could not use git move for dedicated-universal-components. These changes are made in different PR and rebased to this PR.

**Comment by ovh-cds** [comment]
CDS Report installation#17015.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report test#17015.1 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✘

**Review by frenautvh — DISMISSED** [review]

File: packages/manager/modules/billing/src/billing/autoRenew/agreements/details/user-agreements-details.controller.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+
+export default [
```
Comment (cbourgois) [inline packages/manager/modules/billing/src/billing/autoRenew/agreements/details/user-agreements-details.controller.js:4]: Maybe we could use `/* @ngInject */` on this function (it requires to rename some parameter `Service` should be renamed `UserAccountServicesAgreements`...)

File: packages/manager/modules/billing/src/billing/autoRenew/agreements/user-agreements.controller.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import get from 'lodash/get';
+
+export default [
```
Comment (cbourgois) [inline packages/manager/modules/billing/src/billing/autoRenew/agreements/user-agreements.controller.js:3]: Same here.

**Comment by frenautvh** [comment]
@radireddy anything planned regarding this PR ?
it would be great to have billing section as standalone module, thanks!

**Comment by antleblanc** [comment]
@radireddy I think this work should be re-do/rescheduled since #4433 made some good progress recently.

In addition, #3780 has been merged and has introduced several conflicts.

What's your advice? Please let me know.
Thank you!

**Comment by antleblanc** [comment]
@radireddy According to the state of this Pull Request, do you think that can we close it?

**Comment by radireddy** [comment]
This is not required any more because Florian has made the code changes in different PR

branchhttps://github.com/ovh/manager/pull/5028 > feat(dedicated): account section lazy load #5028
https://github.com/ovh/manager/pull/5156 > Feat/billing standalone app
https://github.com/ovh/manager/pull/5121 > refactor(dedicated): move billing section into module)

**Pull request closed without merging**